db/compaction/compaction_picker_level.cc

```diff
@@ -417,6 +423,52 @@ uint32_t LevelCompactionBuilder::GetPathId(
   return p;
 }
 
+bool LevelCompactionBuilder::TryExtendNonL0TrivialMove(int start_index) {
```

Comment:
the same comment as https://github.com/facebook/rocksdb/pull/10188#discussion_r904259437
It might be better to get trivial move files from `FilesByCompactionPri()`.

Reply:
Perhaps I miss something, but I thought the code was written so that trivial move files **are** got from `FilesByCompactionPri()`: line 497 fetches the file in the same way as other compactions, which is stored in `index` and is passed into this function.